manubot/process/metadata.py

```diff
@@ -157,3 +157,53 @@ def get_manuscript_urls(html_url: Optional[str] = None) -> dict:
             + "\n".join(x.url for x in response.history + [response])
         )
     return urls
+
+
+def get_software_versions():
```

Comment:
Once this is stable, can you please add some light documentation for the auto-generated docs?